functions/Get-DbaAgentJob.ps1

```diff
@@ -52,6 +60,8 @@ FUNCTION Get-DbaAgentJob {
 		$SqlCredential,
 		[object[]]$Job,
 		[object[]]$ExcludeJob,
+		[ValidateSet('True', 'False')]
+		[string]$Enabled,
```

Comment:
This should be a bool value, or maybe just a switch. We should not have them type in the word but rather use the variables of `$true` or `$false`. 

However, another point of view is a parameter named `IncludeDisabled` and then by default, the command only returns enabled jobs. If the user wants to include disabled jobs, then they provide the switch. 

Thoughts from @potatoqualitee  ?